src/app/operations/auth.rs

```diff
@@ -3,15 +3,53 @@ use anyhow::{Context, Result};
 use crate::app::{file_manager, App};
 
 impl App {
-    pub async fn user_login(&self, phone: Option<String>, code: String, email: bool) -> Result<()> {
+    pub async fn user_login(
+        &self,
+        phone: Option<String>,
+        code: String,
+        email: bool,
+        two_fa_code: Option<String>,
+    ) -> Result<()> {
```

Comment:
we should not be importing something that overrides a std-lib type.
Best practice is to always write `anyhow::Result` instead of `use anyhow::Result`